framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -3499,6 +3513,170 @@ private boolean isListForRepeatedAnnotationImplementation(DeclaredType annotatio
         return result;
     }
 
+    /**
+     * Whether or not the {@code annotatedTypeMirror} has an implicit qualifier parameter.
```

Comment:
The `implicit` in this comment is not meant as the opposite of `explicit` in the method names below?
Should it be `class qualifier parameter`? Or `implicit or explicit`?